tests/cli/test_compose_sanity.sh

```diff
@@ -22,6 +22,30 @@ rlJournalStart
     rlPhaseEnd
 
     rlPhaseStartTest "compose start"
+        UUID=`$CLI compose start example-http-server ami`
+        rlAssertEquals "exit code should be zero" $? 0
+        UUID=`echo $UUID | cut -f 2 -d' '`
+
+        #wait for the build to properly start
+        sleep 20
+    rlPhaseEnd
+
+    rlPhaseStartTest "compose info"
+        if [ -n "$UUID" ]; then
+            rlRun -t -c "$CLI compose info $UUID | egrep 'RUNNING|WAITING'"
+        else
+            rlFail "Compose UUID is empty!"
+        fi
+    rlPhaseEnd
+
+    rlPhaseStartTest "cancel compose"
+        `$CLI compose cancel $UUID`
```

Comment:
should execute via `rlRun -t -c` if this returns exit code 0. As-is it doesn't make sense to quote in backticks b/c exit code is not checked.

Reply:
This returns and exit code 0 only if I cancel it right after starting the compose. If I wait for ~20 seconds the cancel takes longer and does not exit with 0. I can make the cancel either way but I am not sure which approach covers the scenario better.

Reply:
Cancelling right after in many cases will happen before the compose has even started (e.g. it is still WAITING). The original bug report is about cancel hanging when there is a running compose so this is what we want to cover.  

If I understand the script correctly the current implementation will wait until the status becomes RUNNING  and then cancel it, which will take some time to kill anaconda and cleanup files from disk. This is what you want.

The fact that return code is not 0 is a separate issue.